modules/tinymce/changelog.txt

```diff
@@ -12,6 +12,7 @@ Version 5.6.0 (TBD)
     Added new `emoticons_database` setting to configure which emoji database to use #TINY-6021
     Added new `name` field to the `style_formats` setting object to enable specifying a name for the format. #TINY-4239
     Changed `readonly` mode to allow hyperlinks to be clickable #TINY-6248
+    Fixed missing `TableModified` events for `mceTableRowType` and `mceTableCellType` commands #TINY-6629
```

Comment:
This shouldn't be here, as the `TableModified` event change has never been released.

- [ ] Remove this changelog entry